src/main/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionController.java

```diff
@@ -177,9 +178,11 @@ private void validateTranscriptionRequestValues(TranscriptionRequestDetails tran
                 OffsetDateTime requestStartDateTime = transcriptionRequestDetails.getStartDateTime();
                 OffsetDateTime requestEndDateTime = transcriptionRequestDetails.getEndDateTime();
                 if (requestStartDateTime != null && requestEndDateTime != null) {
-                    boolean validTimes = hearing.getMediaList().stream().anyMatch(m -> m.getStart().isBefore(
-                        requestStartDateTime) && m.getStart().isBefore(requestEndDateTime)
-                        && m.getEnd().isAfter(requestStartDateTime) && m.getEnd().isAfter(requestEndDateTime));
+                    boolean validTimes = hearing.getMediaList().stream().anyMatch(
```

Comment:
Shouldn't this sort of method go in a service rather than a controller? And then unit tests can be written to test the various scenarios rather than integration test?